guides/common/modules/proc_adding-gce-connection.adoc

```diff
@@ -0,0 +1,73 @@
+[id="adding-gce-connection_{context}"]
+= Adding a Google Compute Engine Connection to {ProjectServer}
+
+Use this procedure to add a GCE connection to {ProjectServer}'s compute resources.
```

Comment:
```suggestion
Use this procedure to add a GCE connection to {ProjectServer} compute resources.
```

Reply:
The others all say "Foreman server's compute resources" too - I can change the sentence to avoid the possessive, but just removing the "'s" would break the grammar a bit.